aiidalab_qe/setup_codes.py

```diff
@@ -281,6 +277,10 @@ def _update(self, change):
 
             if self.error or self.installed:
                 self._progress_bar.value = 1.0
+            elif self.busy:
+                self._progress_bar.value = ProgressBar.AnimationRate(1.0)
```

Comment:
What is the rate equal to `1.0` means? Can you add a comment in the code?

Reply:
Sure, would that be sufficient?
```suggestion
                # Start progress bar animation at constant rate (1.0).
                self._progress_bar.value = ProgressBar.AnimationRate(1.0)
```

Reply:
Hmm, I still don't get the intuitive idea of what the `1.0` mean and in what unit?